test_runner/regress/test_signed_char.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+from fixtures.neon_fixtures import NeonEnv
+
+EXPECTED_OUT_PATH = os.path.join(
+    os.curdir, "test_runner", "regress", "data", "test_signed_char.out"
+)
```

Comment:
There is a pytest fixture available. I think it's like `base_dir` or something. You would use it instead of os.curdur.